infra/es_cluster.tf

```diff
@@ -205,10 +230,17 @@ docker run -d \
            -e TELEMETRY_ENABLED=false \
            docker.elastic.co/kibana/kibana:7.13.2
 
+docker run -d \
+           -p 9000:9000 \
+           --link es \
+           --name cerebro \
+           lmenezes/cerebro:0.9.4
```

Comment:
This is a small UI that lets me keep an eye on the health of the cluster. I used to run that from my machine, but Docker is currently not working for me so I haven't kept an eye on this for a week or so. (And of course that when the cluster crashed.) Hopefully in the future this will allow me to notice when things start going wrong a little bit earlier.